### DISCUSSION TITLE
Is it still possible to use apc(u) with magento lts

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#1795] [Questions & Answers]
Is it still possible to use apc(u) with magento lts

**Comment by Flyingmana** [comment]
good question, the apc cache backend should still work, although I did not try it out with apcu yet.
It might be possible, that we need to modify/extend some things to align with changed function names in apcu. Please let us know if you notice some issues there.

**Reply by f1-outsourcing** [reply]
I don't think it is working. I just installed some wordpress plugin to see if my apcu php-fpm setup is correctly configured, and I see cache hits/misses increasing when I open the wordpress site. However the magento seems to do nothing. Tried disabling/enabling Cm_RedisSession and Zend Full Page Cache, but nothing.
I even tried putting the config in different places of local.xml, and also with backend name apcu

```
<config>
    <cache>
        <backend>apc</backend>
        <prefix>xxx3</prefix>
    </cache>
    <global>
        <cache>
            <backend>apc</backend>
            <prefix>xxx</prefix>
        </cache>
        <install>
```

"APCu is the official replacement for the outdated APC extension. APC provided both opcode caching (opcache) and object caching. As PHP versions 5.5 and above include their own opcache, APC was no longer compatible, and its opcache functionality became useless. The developers of APC then created APCu, which offers only the object caching (read "in memory data caching") functionality (they removed the outdated opcache)."[1]


[1]
https://www.php.net/manual/en/book.apcu.php

**Reply by f1-outsourcing** [reply]
@Flyingmana 

Hi Daniel, you have an idea if this is a lot of work?

**Reply by f1-outsourcing** [reply]
Hi @Flyingmana Daniel, you have an idea if this is a lot of work? Afaik this could be quite interesting to have, because enabling this could considerably speed up magento. Or have most people already diverted to other techniques?

**Comment by f1-outsourcing** [comment]
Update: I changed the apc_ functions to apcu_ in these files ./lib/Zend/Cache/Backend/Apc.php and ./app/code/core/Mage/Core/Model/Cache.php. And extension_loaded('apc') to extension_loaded('apcu')

Now I am getting a lot of entries in the apc cache, with prefix_ADMIN_NOTI.. prefix_CORE_CACH.. prefix_DB_PDO all straight away with 150 hits so that looks very nice.

However  I do not really get where this is being enabled in the admin. In config->advanced->system->external full page cache settings. I have enabled to yes, but the control is set to "Zend Full Page Cache" (no other option available)
Storage configuration for media is set to "Filesystem" before making the above changes this was the only option. After these apcu_ changes there is a 2nd option database.
Anyway I would expect somewhere to see APC.
